chickadee/processes/wps_bccaq.py

```diff
@@ -0,0 +1,147 @@
+from pywps import Process, ComplexOutput, LiteralInput, FORMATS
+from pywps.app.Common import Metadata
+from netCDF4 import Dataset
+import os
+
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level
+from chickadee.utils import logger, set_r_options, get_ClimDown, get_doParallel
+
+
+class BCCAQ(Process):
+    """Bias Correction/Constructed Analogues with Quantile mapping reordering:
+    Full statistical downscaling of coarse scale global climate model (GCM)
+    output to a fine spatial resolution"""
+
+    def __init__(self):
+        self.status_percentage_steps = {
+            "start": 0,
+            "process": 10,
+            "build_output": 95,
+            "complete": 100,
+        }
+
+        inputs = [
+            LiteralInput(
+                "gcm_file",
+                "GCM NetCDF file",
+                abstract="GCM simulations in NetCDF format",
+                data_type="string",
+            ),
+            LiteralInput(
+                "obs_file",
+                "Observations NetCDF file",
+                abstract="high-res gridded historical observations",
+                data_type="string",
+            ),
+            LiteralInput(
+                "var",
+                "Variable to Downscale",
+                abstract="Name of the NetCDF variable to downscale",
+                data_type="string",
+            ),
+            LiteralInput(
+                "end_date",
+                "End Date",
+                abstract="Defines the end of the calibration period",
+                default="2005-12-31",
+                data_type="string",
+            ),
```

Comment:
It looks like there may be a [`date`](https://pywps.readthedocs.io/en/latest/api.html#pywps.inout.literaltypes.LITERAL_DATA_TYPES) input option. Perhaps we could explore using that here?

Reply:
The R function that uses it takes a string, not sure if that effects this?

Reply:
In that case this should be alright. Perhaps show the format in the abstract? Either that or we may want to consider some sort of check (if `climdown` doesn't already do one).